### PR TITLE
Support `"parameters": None`

### PR DESCRIPTION
**Pull request by sophieherrmann**

### CODE REVIEW

File: src/openeo_pg_parser/translate.py [hunk 1]
```diff
@@ -292,7 +292,7 @@ def translate_process_graph(pg_filepath, process_defs=None, parameters=None):
 
     # remove first layer of the process graph
     parameters = {} if parameters is None else parameters
-    if "parameters" in process_graph.keys():
+    if "parameters" in process_graph.keys() and process_graph.get("parameters"):
```
Comment (claxn) [inline src/openeo_pg_parser/translate.py:295]: Could you please change  `if "parameters" in process_graph.keys() and process_graph.get("parameters"):` to  `if process_graph.get("parameters"):`? `.get` returns `None` anyway if the key is not present in the dictionary.

File: tests/test_translate.py [hunk 2]
```diff
@@ -17,6 +17,13 @@ def test_translate_process_graph(self):
         print(graph)
         assert True
 
+    def test_translate_process_graph_none(self):
```
Comment (claxn) [inline tests/test_translate.py:20]: How about naming this to `test_translate_process_graph_no_params` or `test_translate_process_graph_none_params`? A process graph is given, but no parameters.

**Review by claxn — CHANGES_REQUESTED** [review]
Some minor comments. Thanks for adding a test too!

File: src/openeo_pg_parser/translate.py (hunk 1 above)
Comment (sophieherrmann) [reply src/openeo_pg_parser/translate.py]: Good point I'll change this

File: tests/test_translate.py (hunk 2 above)
Comment (sophieherrmann) [reply tests/test_translate.py]: Sure I can change this

**Review by claxn — APPROVED** [review]
Thank you for considering my comments!

**Pull request merged**